### DISCUSSION TITLE
HandshakeException

### DISCUSSION THREAD

**Discussion by saifAlkaraawy** [supabase/supabase#26975] [Questions]
![Capture](https://github.com/supabase/supabase/assets/50282354/1b850888-b694-4b37-9a9b-0617dc8ccd2f)
I have problem in windows flutter  app  , how to solve this problem .

HandshakeException: Handshake error in client (OS Error: 
	CERTIFICATE_VERIFY_FAILED: unable to get local issuer certificate(../../flutter/third_party/boringssl/src/ssl/handshake.cc:393))

**Comment by dshukertjr** [comment]
The error is most likely caused by the application not having internet access. Make sure you can make an HTTP request from the app.
